### PR TITLE
2986 target essentiality

### PR DESCRIPTION
**Pull request by remo87**
- add target essentiality output
- format files

### CODE REVIEW

File: src/main/scala/io/opentargets/etl/backend/target/Target.scala [hunk 1]
```diff
@@ -269,28 +281,33 @@ object Target extends LazyLogging {
...
-      .join(lookup_table, lookup_table("symbol") === geneEssentiality("targetSymbol"), "inner")
+      .join(lookup_table,
+            lookup_table("approvedTarget") === geneEssentiality("targetSymbol"),
+            "inner"
+      )
       .drop(lookup_table.columns.filter(_ != "ensgId"): _*)
       .drop("targetSymbol")
 
-    val targetEssentialityGroupById = targetEssentialityWithEnsgId
+    val test01 = targetEssentialityWithEnsgId
```
Comment (mbdebian) [inline src/main/scala/io/opentargets/etl/backend/target/Target.scala:297]: Would you mind refactoring this?
It may not look right

**Review by mbdebian — CHANGES_REQUESTED** [review]
@remo87 , would you mind looking at the comments in the code?
Thanks!

File: src/main/scala/io/opentargets/etl/backend/target/Target.scala (hunk 1 above)
Comment (remo87) [reply src/main/scala/io/opentargets/etl/backend/target/Target.scala]: It's been applied

**Review by mbdebian — APPROVED** [review]
Thanks @remo87 , nice work!

**Pull request merged**